app/views/talks/_talk.html.erb

```diff
@@ -9,6 +9,11 @@
 
       <h4><%= render_markdown_as_html "#{talk.title.titleize}, by #{talk.member.name}" %></h4>
 
+      <% if talk.event&.starts_at&.past? && talk.youtube_id.present? %>
+        <iframe src="https://www.youtube.com/embed/<%= talk.youtube_id %>?rel=0&autoplay=0" frameborder="0" allowfullscreen width="640" height="360"></iframe>
```

Comment:
This pull-request is pretty simple and very good. The only suggestion I would make is create a helper for the youtube tag